### DISCUSSION TITLE
Windows 3.11 on QEMU 5.2.0 - Alan Pope's blog

### DISCUSSION THREAD

**Discussion by giscus** [popey/popey.com-blog#74] [General]
# Windows 3.11 on QEMU 5.2.0 - Alan Pope's blog

This is mostly an informational PSA for anyone struggling to get Windows 3.11 working in modern versions of QEMU. Yeah, I know, not exactly a massively viral target audience.
Anyway, short answer, use QEMU 5.2.0 from December 2020 to run Windows 3.11 from November 1993.
An innocent beginning I made a harmless jokey reply to a toot from Thom at OSNews, lamenting the lack of native Mastodon client for Windows 3.

https://popey.com/blog/2024/09/windows-3.11-on-qemu-5.2.0/

**Comment by tbodt** [comment]
You should git bisect it! I'm sure you can find the git commit in qemu that broke it and get this fixed, making future qemu archaeology unnecessary.
